cloud-agnostic/core/src/DIContainer.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Is this disable required? Don't see "any" used here.

Reply:
Removed unnecessary eslint disables. Also noticed that I had left a lot of "reflect-metadata" imports in tests, that are no longer relevant, since there's not singular import file for that.